app/scripts/metamask-controller.js

```diff
@@ -1561,7 +1563,7 @@ export default class MetamaskController extends EventEmitter {
         },
       },
       env: {
-        isAccountSyncingEnabled: isManifestV3,
+        isAccountSyncingEnabled: !isProduction() && isManifestV3,,
```

Comment:
This extra comma is causing the unit test failure:
```suggestion
        isAccountSyncingEnabled: !isProduction() && isManifestV3,
```